sets.json

```diff
@@ -259,82 +152,6 @@
       "wikipedia_420.y4m"
     ]
   },
-  "objective-1-fast-65": {
```

Comment:
We do have this one, I would like to leave it for now as it was historically used several times.

Reply:
When was it last used and can it be uploaded to https://media.xiph.org/sets/?

Reply:
It was used for AV2 baseline selection (libaom vs SVT-AV1). I can upload it there.